src/request-params.ts

```diff
@@ -24,194 +17,79 @@ type PaginationQueryParams = {
   next?: string;
 };
 
-const pathParams = Lens.fromNullableProp<APIGatewayEvent>()(
-  "pathParameters",
-  {}
-);
-type PathParams = NonNullable<
-  APIGatewayEvent["pathParameters"] & {
-    hash?: string;
-    term?: string;
-    address?: string;
+export const extractHashOrdinal = (term) => {
```

Comment:
`term` can be "latest" here. How are we handling that? I don't see additional handling for that in the controllers.